x-pack/legacy/plugins/fleet/server/libs/policy.test.ts

```diff
@@ -3,10 +3,10 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { Policy, Status } from '../../../ingest/server/libs/adapters/policy/adapter_types';
 import { FrameworkUser, internalAuthData } from '../adapters/framework/adapter_types';
 import { AgentPolicy } from '../repositories/policies/types';
 import { PolicyLib } from './policy';
+import { Status, Policy } from '../../../ingest/server/libs/types';
```

Comment:
if we export Status from [here](https://github.com/elastic/kibana/pull/52061/files#diff-b834b42580f51b467e14fd3108203566R7) too, then we can avoid traversing into another plugin here. as fleet will be using ingest types often, I'd like to try to export ingest types from one place inside fleet (i.e.  from `/common/types/domain_data.ts`) so that most other fleet code doesn't have to traverse into ingest. what do you think?